src/main/java/core/basesyntax/service/FruitFIleWriterImpl.java

```diff
@@ -0,0 +1,19 @@
+package core.basesyntax.service;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.nio.file.Path;
+
+public class FruitFIleWriterImpl implements FruitFileWriter {
+    @Override
+    public void writeToFile(String data, Path filePath) {
```

Comment:
```suggestion
    public void writeToFile(String data, String filePath) {
``` 
лучше используй здесь String , так тебе не нужно будет делать valueOf()